### PR TITLE
Base image for enabling work with nvidia GPUs

### PR DESCRIPTION
**Pull request by leafty**
* Added an image with cuda 9.2, for GPU enabled notebooks
* Made travis run chartpress to validate all image builds

### CODE REVIEW

**Comment by rokroskar** [comment]
Looks good, have you tried it?

**Comment by leafty** [comment]
yes, with this appended (for tf):
```
RUN conda install --quiet --yes \
    'tensorflow-gpu=1.11*' && \
    conda clean -tipsy && \
    fix-permissions $CONDA_DIR && \
    fix-permissions /home/$NB_USER
```
then with:
```yaml
apiVersion: v1
kind: Pod
metadata:
  name: test-jmt-gpu
  namespace: renku
spec:
  restartPolicy: OnFailure
  containers:
    - name: jupyter-notebook
      image: "registry.limited.renku.ch/leafty/test:base"
      command: ["jupyter-notebook"]
      resources:
        limits:
          nvidia.com/gpu: 1
```

It gives:
image 2

File: singleuser/cuda.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# From: https://gitlab.com/nvidia/cuda/blob/ubuntu18.04/9.2/base/Dockerfile
+USER root
+RUN apt-get update && apt-get install -y --no-install-recommends gnupg2 curl ca-certificates && \
+    curl -fsSL https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1710/x86_64/7fa2af80.pub | apt-key add - && \
+    echo "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1710/x86_64 /" > /etc/apt/sources.list.d/cuda.list && \
+    echo "deb https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64 /" > /etc/apt/sources.list.d/nvidia-ml.list && \
+    apt-get purge --autoremove -y curl && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV CUDA_VERSION 9.2.148
```
Comment (rokroskar) [inline singleuser/cuda.Dockerfile]: Make this an ARG?

Comment (leafty) [reply singleuser/cuda.Dockerfile]: The thing is taken directly from nvidia: https://gitlab.com/nvidia/cuda/blob/ubuntu18.04/9.2/base/Dockerfile

Don't know if that's the only change with other cuda versions.

Comment (rokroskar) [reply singleuser/cuda.Dockerfile]: I see, unfortunately there's a new Cuda version every few weeks

**Comment by rokroskar** [comment]
Is there a reason you haven't added this to chart press?

**Comment by leafty** [comment]
oops

**Comment by leafty** [comment]
@rokroskar up for review

**Pull request merged**